src/items/qnitelinepainter.cpp

```diff
@@ -62,8 +64,13 @@ void QniteLinePainter::paint(QNanoPainter *painter) {
   if (pen.fill.isValid()) {
     painter->beginPath();
     painter->moveTo(m_xs.at(0), m_baseline);
-    for (auto i = 0; i < dataSize; ++i) {
-      painter->lineTo(m_xs.at(i), m_ys.at(i));
+    painter->lineTo(m_xs.at(0), m_ys.at(0));
+    for (auto i = 1; i < dataSize; ++i) {
+        if (m_drawStepped)
```

Comment:
Please use `clang-format` to reformat your code. There is a `.clang-format` at the root of the project which will apply proper styles to the code